mobile/src/components/common/text/text.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC, ReactElement } from 'react';
+import { Text, TextStyle } from 'react-native';
+
+import { AppFontFamily } from '~/common/enums/ui/ui';
```

Comment:
Don't forget to re-export it
```suggestion
import { AppFontFamily } from '~/common/enums/enums';
```